tests/src/test/scala/scalaz/MapTest.scala

```diff
@@ -640,6 +794,120 @@ object MapTest extends SpecLite {
       val f = (k: Int, a: String, result: String) => result + "(" + k.toString + ":" + a + ")"
       fromList(List(5 -> "a", 3 -> "b")).foldrWithKey("Map: ")(f) must_== "Map: (5:a)(3:b)"
     }
+
+    "foldMapWithKey" ! forAll { a: Byte ==>> Byte =>
+      val intAddition = new Monoid[Int] {
```

Comment:
Oh, it actually is. I didn't explicitly provide `intAddition` to `foldMapWithKey` as an argument as the signature is like below.
`def foldMapWithKey[C](f: (A, B) => C)(implicit F: Monoid[C]): C =`

The following might be better for this case? WDYT?
```
a.foldMapWithKey(f)(intAddition) must_=== res
```